### DISCUSSION TITLE
Furqan Dubai Report

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#191] [General]
REF: [Tally Proposal](https://www.tally.xyz/gov/dcdao/proposal/50559279770185093185630632312686269587339353772333996266645150746493617082787) 

I am making this post event report to share what was done - and share expenses and the value add details. It starts here and have an expansive summary detailing everything from the expenses to recipets to people met - if someone catches your eye - please feel free to lmk and I will make a connection!

You might be wondering why the title of this report isn’t “Furqan Cosmoverse Report” we will get into that in a minute but I had an amazing time in Dubai meeting with various key leaders and getting Darwinia out there! I want to emphasize how I had people gasping at the fact that Darwinia still exists, and how they thought we were a... (continued [here](https://docs.google.com/document/d/1DwL7geFF2JdUWbSkrNTXXDTiARyOLqZDd3pN6F5prMo/))

**Comment by SasoLithops** [comment]
I must congratulate Furqan on what he went through and stuck with "I am going, no matter what". He had huge problems with visa, got it in literary last hours, but he did! How he met and talked with so many people is fantastic. We already have so many active TG groups from new acquaintances. Huge extra congrats on my part for getting Osmosis and Kepler wallet. Osmosis is THE catch imho.

This report is exactly how good and healthy DAO works. This HAS TO become common practice for future events visited by RING DAO members.   It has all the transparency it needs, costs callculation and subjective opinion on what was achived which I find to be the best to move things forward.

Also I could not agree we need to go out more and to TIER 1 and TIER 2 conferences, where we meet new people, create TG groups where  appropriate DAO members discuss this further. RING DAO members are doing great in new established groups and Cosmos eco teams look interested. I would recommend we go to at least two conferences per each quarter. This is starting to show to be best marketing move.
BUT WE ALL MUST DO EVEN BETTER RESEARCH BEFORE GOING TO THIS CONFERENCES, SO WE KNOW WHICH TEAMS TO TARGET. I wish I did my part even better in picking more then few as Furqan did amazing asking me who you need next:-)

As for the ammount going to this conferences I recommend we dedicate 5000$ for TIER 1 conference( main token/project/company conferences) visitations and 3500$ for TIER 2 conferences(secondary token/project/company conferences). I also don't recommend any sponsorships at this time nor booths as that goes to over 50000$ which we could not afford with RING liquidity atm. Whatever is left unused gets refunded to RING DAO account. 
We must continue to mingle among people and I find Furqan to be the one to lead this. As he is very open person, speaks English well and has a good knowledge of our tech to attend conferences. We must start promoting RING DAO as I must sadly agree most think Darwinia is dead as we have not been anywhere important in last few years. I also think we must distance ourselves from being marketed as Polkadot parachain. We are so much more then this.

As for conferences I too have found from others before, these conferences are fun and partnerships are discussed later after the conference. Imho onferences are for reaching out to people, certain projects you plan to reach and meeting new people and having fun as down the line you never know in crypto who will be the next best thing.

At the end but still important I am asking for us to vote on giving Furqan back to what he spent out of his own pocket. He mentiones this in pending payments. I think he deserves this. It could be paid from what was left from Q3.

Let's grow on together showing what RING DAO can do in crypto space.

**Reply by nblogist** [reply]
THANK YOU SO MUCH @SasoLithops we couldn't have kicked it off without your support!

I appreciate you recognizing my efforts! :heart:  I can't wait for Darwinia to go back to mainstream!

**Comment by nblogist** [comment]
CC @hackfisher @wuminzhe @furoxr
